### PR TITLE
{vis}[foss/2017b] ITK-SNAP v3.6.0

### PR DESCRIPTION
**Pull request by murfreesboro**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by murfreesboro** [comment]
This software relies on an include file itkKernelFunction.h (from dependent library of ITK). This file exists in the version 4.8.*; for the newer ITK like 4.12 etc. this file does not exist anymore. So there's compiling dependency on a given old library. Therefore to directly compile it is not doable without this old library.

**Comment by murfreesboro** [comment]
Test report by @murfreesboro
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
dev806.vampire - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.5
See https://gist.github.com/e56bbbd3c55b830ea19ff75e7218c724 for a full test report.

File: easybuild/easyconfigs/i/ITK-SNAP/ITK-SNAP-3.6.0-foss-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+# The foss toolchain is needed because the loading of binary needs the libGLU
+# therefore the toolchain is not dummy
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = ['https://sourceforge.net/projects/itk-snap/files/itk-snap/3.6.0']
+sources = ['itksnap-%(version)s-20170401-Linux-x86_64-qt4.tar.gz']
+checksums = ['b0818d0de4ae8295e514d02f5fccea9c144a56f088c40b2f70a6a78616954210']
+
+dependencies = [
+    ('libpng', '1.2.58', '', True),
```
Comment (boegel) [inline easybuild/easyconfigs/i/ITK-SNAP/ITK-SNAP-3.6.0-foss-2017b.eb]: @murfreesboro You should bump this to `1.6.32` to avoid introducing conflicts...

Comment (vanzod) [reply easybuild/easyconfigs/i/ITK-SNAP/ITK-SNAP-3.6.0-foss-2017b.eb]: @boegel This installation uses a pre-compiled binary so there is no way to link statically against `libpng/1.2.x`. I vote for closing it.

Comment (boegel) [reply easybuild/easyconfigs/i/ITK-SNAP/ITK-SNAP-3.6.0-foss-2017b.eb]: OK, closing then...

**Pull request closed without merging**